osgpkitools/osg-cert-revoke

```diff
@@ -3,7 +3,7 @@
 import sys
 import urllib
 import httplib
-import M2Crypto
+from M2Crypto import httpslib, SSL
```

Comment:
Does this work?  You're not importing `M2Crypto` into the namespace here anymore, so when it is used later I'd expect to get a `NameError`